include/gauge_field_order.h

```diff
@@ -367,6 +377,7 @@ namespace quda {
 
     template <typename Float, int nColor, typename storeFloat>
     struct Accessor<Float, nColor, QUDA_QDP_GAUGE_ORDER, storeFloat> {
+      static constexpr bool is_aos = false;
```

Comment:
Should be `= true`, if `AoS` is with regards to color/complex.

The rough difference between `QUDA_MILC_GAUGE_ORDER` and `QUDA_QDP_GAUGE_ORDER` is whether or not the direction of the gauge link is slower than coordinate or not, in all cases color/complex is still the fastest index.

Reply:
Update: digging deeper into this, I see that the implementation requires `QUDA_MILC_GAUGE_ORDER` in particular. That's fine, but that means we should update this `constexpr` from `is_aos` to `is_mma_compatible` or something like that: `is_aos = false` for `QUDA_QDP_GAUGE_ORDER` is factually wrong.

Reply:
Well `QUDA_QDP_GAUGE_ORDER` is compatible with MMA as well :( : we will add `wrap` methods for it as well then?

Reply:
Aye, please do. I saw there are some explicit checks for the field order being `QUDA_MILC_GAUGE_ORDER`. If so desired could that be extended to both that or `QUDA_QDP_GAUGE_ORDER` without correctness issues?

I can understand that there's no use for it, if anything to avoid extra compiles, so this is just a technical correctness question.

Reply:
Nah after some thought I will just change `is_aos` to `is_mma_compatible`. Apart from increasing the compile time, adding a bunch of (currently) redundant `wrap`s is confusing. If in the future `QUDA_QDP_GAUGE_ORDER` is needed for MMA, adding these should be trivial.